src/Sulu/Component/Content/Compat/Structure/StructureBridge.php

```diff
@@ -449,6 +450,18 @@ public function getNodeName()
         return $this->getDocument()->getTitle();
     }
 
+    public function getNodePath(): ?string
+    {
+        if ($this->document instanceof RedirectTypeBehavior &&
+            RedirectType::INTERNAL == $this->document->getRedirectType() &&
+            null !== $this->document->getRedirectTarget()
+        ) {
+            return $this->getDocument()->getRedirectTarget()->getPath();
```

Comment:
Currently the `path` attribute did not resolve the redirect, did it? If we are going to introduce that one as a replacement for the old `path`, shouldn't it behave the same?

Reply:
And btw, in the [`ContentMapper`](https://github.com/sulu/sulu/pull/5273/files#diff-2a74d79cea8c966c3ad6a835206c6c3bR733) it also does not seem to be resolved, am I right?